### PR TITLE
Updated README.md to include a fix for line endings

### PR DESCRIPTION
**Pull request by zandercymatics**
#  #

## 🗣 Description ##
Updated the readme to include a fix for windows machines...

On windows, you cannot run manage.py (or other related files) as they are using the wrong file endings. It basically doesn't know where to look.

The fix to this is to add .gitattribute locally (or globally) and rebuild your project, so that it reverts to standard line-endings on your machine (but not elsewhere)

## 💭 Motivation and context ##
To inform future developers using windows machines

### CODE REVIEW

File: docs/developer/README.md [hunk 1]
```diff
@@ -18,6 +18,9 @@ If you're new to Django, see [Getting Started with Django](https://www.djangopro
 
 Visit the running application at [http://localhost:8080](http://localhost:8080).
 
+Note: If you are using Windows, you may need to change your [line endings](https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings).
```
Comment (abroddrick) [inline docs/developer/README.md:21]: I would add a reason why here so that the issue with the line endings is clear.

**Comment by zandercymatics** [comment]
@abroddrick Good point... I've added a bit more detail there so hopefully its more descriptive now

**Review by abroddrick — APPROVED** [review]
LGTM thanks for the changes!

**Review by abroddrick — APPROVED** [review]

**Review by abroddrick — APPROVED** [review]

**Comment by abroddrick** [comment]
@zandercymatics don't forget to merge

**Pull request merged**